synedrion/src/sessions/type_erased.rs

```diff
@@ -18,15 +18,16 @@ use crate::rounds::{
 };
 
 pub(crate) fn serialize_message(message: &impl Serialize) -> Result<Box<[u8]>, LocalError> {
-    bincode::serialize(message)
+    bincode::serde::encode_to_vec(message, bincode::config::legacy())
```

Comment:
So I wasn't sure if we actually needed to care about compatibility with `bincode@1.x` here, so I went with the legacy encoding. If we don't care we can use `bincode::config::standard()` instead

Reply:
Not really, it just needs to be consistent with the decoding counterpart.